roc/version_test.go

```diff
@@ -0,0 +1,86 @@
+package roc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersion(t *testing.T) {
+	v := Version()
+	require.NotZero(t, v.Native.Major+v.Native.Minor+v.Native.Patch)
+	require.NotZero(t, v.Bindings.Major+v.Bindings.Minor+v.Bindings.Patch)
+	require.Equal(t, v.Native.Major, v.Bindings.Major)
+	require.GreaterOrEqual(t, v.Bindings.Minor, v.Native.Minor)
+}
+
+func Test_parseVersion(t *testing.T) {
```

Comment:
nit: I suggest to rename tests to something like this:

* TestVersion => TestVersion_Get
* Test_parseVersion => TestVersion_Parse

For consistency with other tests, where we name them `Test<File>_<Component>`.